packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -230,6 +215,10 @@ export default mixins(
 			sessionId: {
 				type: String,
 			},
+			nodeType: {
+				type: Object,
```

Comment:
can add type like this 
```suggestion
				type: Object as PropType<INodeTypeDescription>,
```